### PR TITLE
[script.trakt] 3.3.0

### PR DESCRIPTION
**Pull request by Razzeee**
### Description
Here's the new version for matrix.
Two things happend, I ported to matrix and lunatix did some work on PVR detection.

Diff is here:
https://github.com/trakt/script.trakt/compare/3.2.4...master

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

File: script.trakt/.gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+*.pyc
```
Comment (enen92) [inline script.trakt/.gitignore]: remove from submission please

File: script.trakt/.gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+test:
```
Comment (enen92) [inline script.trakt/.gitlab-ci.yml]: same

File: script.trakt/.travis.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+language: python
```
Comment (enen92) [inline script.trakt/.travis.yml]: same

File: script.trakt/addon.xml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+		<language></language>
+		<platform>all</platform>
+		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```
Comment (enen92) [inline script.trakt/addon.xml]: Can you add a SPDX identifier instead of the full textual version?

File: script.trakt/addon.xml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+		<language></language>
```
Comment (enen92) [inline script.trakt/addon.xml]: remove if empty

File: script.trakt/addon.xml [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+		<platform>all</platform>
+		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+		<forum>https://forum.kodi.tv/showthread.php?tid=220547</forum>
+		<website>https://trakt.tv</website>
+		<email></email>
```
Comment (enen92) [inline script.trakt/addon.xml]: remove if empty

File: script.trakt/changelog.txt [hunk 7]
```diff
@@ -0,0 +1,197 @@
+version 3.3.0
```
Comment (enen92) [inline script.trakt/changelog.txt]: remove from submission

File: script.trakt/resources/lib/deviceAuthDialog.py [hunk 8]
```diff
@@ -0,0 +1,59 @@
+import xbmcgui
```
Comment (enen92) [inline script.trakt/resources/lib/deviceAuthDialog.py]: `__init__.py` files are not necessary with python3

File: script.trakt/resources/lib/kodiUtilities.py [hunk 9]
```diff
@@ -0,0 +1,356 @@
...
+def showSettings():
+    __addon__.openSettings()
+
+def getSetting(setting):
+    return __addon__.getSetting(setting).strip()
+
+def setSetting(setting, value):
+    __addon__.setSetting(setting, str(value))
+
+def getSettingAsBool(setting):
```
Comment (enen92) [inline script.trakt/resources/lib/kodiUtilities.py]: should not be needed in newer api versions

File: script.trakt/resources/lib/kodilogging.py [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+            logging.ERROR: xbmc.LOGERROR,
+            logging.WARNING: xbmc.LOGWARNING,
+            logging.INFO: xbmc.LOGINFO,
+            logging.DEBUG: xbmc.LOGDEBUG,
+            logging.NOTSET: xbmc.LOGNONE,
+        }
+        if getSettingAsBool('debug'):
+            try:
+                xbmc.log(self.format(record), levels[record.levelno])
+            except UnicodeEncodeError:
```
Comment (enen92) [inline script.trakt/resources/lib/kodilogging.py]: should not be needed (matrix is py3 only)

File: script.trakt/tests/fixtures/compare_shows_batman_episode_to_add.json [hunk 11]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (enen92) [inline script.trakt/tests/fixtures/compare_shows_batman_episode_to_add.json]: please remove the tests/ folder from submission. Maybe you can think about automating the submissions and add all those files to .gitattributes

File: script.trakt/.gitignore (hunk 1 above)
Comment (Razzeee) [reply script.trakt/.gitignore]: Done

File: script.trakt/.travis.yml (hunk 3 above)
Comment (Razzeee) [reply script.trakt/.travis.yml]: Done

File: script.trakt/addon.xml (hunk 5 above)
Comment (Razzeee) [reply script.trakt/addon.xml]: Done

File: script.trakt/addon.xml (hunk 6 above)
Comment (Razzeee) [reply script.trakt/addon.xml]: Done

File: script.trakt/changelog.txt (hunk 7 above)
Comment (Razzeee) [reply script.trakt/changelog.txt]: Done

File: script.trakt/resources/lib/deviceAuthDialog.py (hunk 8 above)
Comment (Razzeee) [reply script.trakt/resources/lib/deviceAuthDialog.py]: Done

File: script.trakt/resources/lib/kodiUtilities.py (hunk 9 above)
Comment (Razzeee) [reply script.trakt/resources/lib/kodiUtilities.py]: Can I change that in the next version?

File: script.trakt/resources/lib/kodilogging.py (hunk 10 above)
Comment (Razzeee) [reply script.trakt/resources/lib/kodilogging.py]: Can I change that in the next version?

File: script.trakt/tests/fixtures/compare_shows_batman_episode_to_add.json (hunk 11 above)
Comment (Razzeee) [reply script.trakt/tests/fixtures/compare_shows_batman_episode_to_add.json]: Done

File: script.trakt/.gitlab-ci.yml (hunk 2 above)
Comment (Razzeee) [reply script.trakt/.gitlab-ci.yml]: Done

File: script.trakt/addon.xml (hunk 4 above)
Comment (Razzeee) [reply script.trakt/addon.xml]: Fixed

File: script.trakt/resources/lib/kodiUtilities.py (hunk 9 above)
Comment (enen92) [reply script.trakt/resources/lib/kodiUtilities.py]: sure

File: script.trakt/resources/lib/kodilogging.py (hunk 10 above)
Comment (enen92) [reply script.trakt/resources/lib/kodilogging.py]: sure

**Pull request merged**